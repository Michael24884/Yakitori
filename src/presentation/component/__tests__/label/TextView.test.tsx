/**
 * @format
 */
import React from 'react';
import {Text} from 'react-native';
import {TextView} from '@components';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const instance = renderer.create(<TextView />);
  expect(instance.toJSON()).toMatchSnapshot();
});

it('renders with text', () => {
  const instance = renderer.create(<TextView text="test" />);
  const textView = instance.root.findByType(Text);
  expect(textView.props.children).toBe('test');
});
