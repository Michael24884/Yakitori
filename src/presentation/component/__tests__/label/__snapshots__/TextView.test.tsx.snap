// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#4f4f4f",
    }
  }
/>
`;
